lib/opera/operation/instructions/executors/validate.rb

```diff
@@ -14,8 +14,13 @@ def break_condition
           def evaluate_instruction(instruction)
             instruction[:kind] = :step
             dry_result = super
-            add_instruction_output(instruction, dry_result.to_h)
-            result.add_errors(dry_result.errors) unless dry_result.success?
+
+            if dry_result.success?
```

Comment:
we don't need that. I fixed that by adding `.to_h` on the left.

Reply:
but please, double-check. At least it works just fine in HAL

Reply:
https://github.com/Profinda/opera/commit/e5e83ca1b3fce96e4e85e2d100327f038114d131#diff-828218ef941ed5d36e1ab826614301e149b77cbc896665aaae4b327d3bee4449R17

Reply:
doesn't work, unfortunately - https://app.circleci.com/pipelines/github/Profinda/profinda_saas/3934/workflows/dce9fe51-3a10-474f-a72a-c6f0fe05d457/jobs/53381

Reply:
but the error is about `information`

Reply:
no @k-rudy. It works. Problem is that our code is a hack. In step `validate` we expect dry-validation result, not sth else. Here we have custom validator which tries to mimic our intefrace and it breaks. Please update our validator:
https://github.com/Profinda/profinda_saas/blob/master/engines/pro_finda/lib/pro_finda/validator.rb

add line:
```
delegate to_h, to: :errors
```